pygmt/tests/test_grdimage.py

```diff
@@ -73,7 +73,7 @@ def test_grdimage_file():
     return fig
 
 
-@pytest.mark.xfail(
+@pytest.mark.skip(
     reason="Upstream bug in GMT 6.1.1",
     condition=gmt_version <= Version("6.1.1"),
 )
```

Comment:
Can we add a condition to only skip this for macOS?

Reply:
Sometimes, it also crashes for Windows.

Reply:
I thought the Windows test failures are to do with failed documentation builds?

Reply:
Maybe you're right. I can't find Windows failures that were caused by crashes.

Reply:
Done.